bugify.go

```diff
@@ -0,0 +1,72 @@
+package bugify
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+var GitHubAPIURL = "https://api.github.com"
+var HTTPClient = &http.Client{Timeout: 10 * time.Second}
+
+// Issue represents information about an GitHub issue.
+type Issue struct {
+	Title        string
+	Body         string
+	GitHubAPIKey string
+	Repo         string
+}
+
+type apiRequest struct {
+	Title string `json:"title"`
+	Body  string `json:"body"`
+}
+
+type apiResponse struct {
+	ID  int64  `json:"id"`
```

Comment:
Actually, we don't need the ID any more, do we? We had thought of using it to construct the issue URL, but we have that already, so the ID isn't used.